src/components/player-list/index.tsx

```diff
@@ -1,24 +1,28 @@
 import { List } from 'antd'
+import 'antd/dist/antd.css'
```

Comment:
If we need `antd` styling we need to import it in the `tsx` file like this going forward. There are no `.css` files now.

Reply:
That's kind of stupid. They have react components, but they don't include the required CSS automatically?

Reply:
I've fixed this now.